web/src/lib/utils/focus-change.ts

```diff
@@ -0,0 +1,21 @@
+interface Options {
+  onFocusOut?: () => void;
+}
+
+export function focusChange(node: HTMLElement, options: Options = {}) {
```

Comment:
Why not make the callback required and use similar naming to `clickOutside`?

`export function focusOutside(node: HTMLElement, onFocusOut: () => void) {`

Reply:
I was trying to keep the naming generic enough such that if an opposite `onFocusIn` handler was ever needed, the name would still fit. Changing the name to `focusOutside` for consistency works for me :+1: